spec/unit/food_truck_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../../app/food_truck'
+require_relative '../../constants'
+
+module FoodTruck
+  RSpec.describe FoodTruck do
+    describe 'FoodTruck loads data' do
```

Comment:
Does this `describe` block serve a useful purpose?  It looks like the example can move into the outer, parent example group.